backend/internal/server/apis/schedule/pkg/parser/games.go

```diff
@@ -0,0 +1,66 @@
+package parser
+
+import (
+	"fmt"
+	"github.com/jak103/powerplay/internal/server/apis/schedule/pkg/models"
+	"github.com/jak103/powerplay/internal/utils/log"
+	"os"
+	"time"
+
+	"github.com/gocarina/gocsv"
+)
+
+func ReadGames(season string) ([]models.Game, models.SeasonConfig) {
+	log.Info("Reading config file season_config.yml\n")
+
+	seasonConfig, err := SeasonConfig(season)
+	if err != nil {
+		log.Error("Error reading file: %v\n", err)
+	}
+
+	var games []models.Game
+
+	scheduleFile, err := os.OpenFile(fmt.Sprintf("schedule_%s.csv", season), os.O_RDONLY, os.ModePerm)
+	if err != nil {
+		log.Error("Failed to open schedule file: %v\n", err)
+	}
+
+	err = gocsv.UnmarshalFile(scheduleFile, &games)
+	if err != nil {
+		log.Error("Failed to unmarshal schedule file: %v\n", err)
+	}
+
+	log.Info("Read %v games\n", len(games))
+
+	for i := range games {
+		games[i].Start, err = time.Parse("01/02/2006 15:04", fmt.Sprintf("%v %v", games[i].StartDate, games[i].StartTime))
```

Comment:
I am confused why we are parsing this string into a time?

Reply:
The first argument of time.Parse is a format string, is that what you are asking about?

Reply:
> I am confused why we are parsing this string into a time?

Our current game model requires Start to be a time.Time, that is why we are parsing it. I think the idea is that we won't have the parser package when we are done? I'm not sure though

Reply:
no  I am just confused about the significance of 01/02/2006 15:04

Reply:
Oh I guess go just always uses this date to specify the format. https://gobyexample.com/time-formatting-parsing

never seen a format string specified that way but i kinda like it